website/content/upgrading/from_v1_to_v2.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Upgrade from v1 to v2
+---
+
+Dkron v2 brings lots of changes to the previous version. To successfully upgrade from v1 to v2 you need to take care of certain changes:
+
+## Migrating Jobs
+
+To migrate jobs from v1 to v2 they should be exported from the v1 cluster and imported into v2.
```

Comment:
To migrate jobs from v1 to v2, export jobs from the v1 cluster and import them into v2.